node/pkg/p2p/cutover.go

```diff
@@ -0,0 +1,131 @@
+package p2p
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"go.uber.org/zap"
+)
+
+// The format of this time is very picky. Please use the exact format specified by cutOverFmtStr!
+const mainnetCutOverTimeStr = ""
+const testnetCutOverTimeStr = "2024-12-31T23:59:59-0000"
+const devnetCutOverTimeStr = "2022-12-31T23:59:59-0000"
```

Comment:
just to confirm, this means devnet is using quic-v1 after this PR?

Reply:
Right. Had an offline discussion with @evan-gray and we agreed this was okay.